packages/utilities/psammead-storybook-helpers/src/themes.js

```diff
@@ -0,0 +1,15 @@
+import { create } from '@storybook/theming';
+
+const createTheme = props =>
+  create({
+    brandTitle: 'BBC Psammead',
+    brandUrl: 'https://github.com/bbc/psammead',
+    brandImage:
+      'https://user-images.githubusercontent.com/11341355/54079666-af202780-42d8-11e9-9108-e47ea27fddc5.png',
+    ...props,
+  });
+
+export default {
+  light: createTheme({ base: 'light' }),
```

Comment:
Should we call this `mode` instead of `base`? Just thinking

Reply:
`base` is from the storybook-theming api - it's not something we can rename unfortunately

https://storybook.js.org/docs/configurations/theming/#create-a-theme-quickstart